src/components/getStats/getStats.js

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable camelcase */
+import React, { Component } from 'react';
+import '../../assets/styles/components/getStatistics.scss';
+import { connect } from 'react-redux';
+import moment from 'moment';
+import { Table, Badge } from 'react-bootstrap';
+import { callApiThunk } from '../../redux/thunk/index';
+import { allMyrequests } from '../../redux/actions/requestActions';
+import Nav from '../Nav';
+import { Stats, TableHeader } from '../../util/reusableTableHeader';
+
+class numberOfTrips extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      allRequests: [],
+      data: [],
+      upToNow: new Date(),
+      intervel: 0,
+      trips: 0,
+      pending: 0,
+      rejected: 0,
+      startDate: 0,
+    };
+  }
+
+
+  async UNSAFE_componentWillMount() {
+    await this.props.callApiThunk('get', '/trips/myrequest', allMyrequests);
+    this.setState({
+      allRequests: this.props.userStats.allMyRequest,
+      data: this.props.userStats.allMyRequest.filter((trip) => trip.status === 'approved'),
+      pending: this.props.userStats.allMyRequest.filter((trip) => trip.status === 'pending'),
+      rejected: this.props.userStats.allMyRequest.filter((trip) => trip.status === 'rejected'),
+      trips: this.props.userStats.allMyRequest.filter((trip) => trip.status === 'approved').length,
+      startDate: (this.props.userStats.allMyRequest.filter((trip) => trip.status === 'approved').length) ? this.props.userStats.allMyRequest.filter((trip) => trip.status === 'approved')[0].departureDate : 0,
```

Comment:
According to your ternary operator, what does having a startDate of zero mean?

Reply:
That is the first date you started to travel if you haven't done any trip, startDate will be O